lit_llama/model.py

```diff
@@ -5,20 +5,29 @@
 # mypy: ignore-errors
 import math
 from dataclasses import dataclass
+from typing import Optional
 
 import torch
 import torch.nn as nn
 from torch.nn import functional as F
 from typing_extensions import Self
 
+from lit_llama.utils import find_multiple
+
+
 @dataclass
 class LLaMAConfig:
     block_size: int = 2048
     vocab_size: int = 32000
+    padded_vocab_size: Optional[int] = None
     n_layer: int = 32
     n_head: int = 32
     n_embd: int = 4096
 
+    def __post_init__(self):
+        if self.padded_vocab_size is None:
+            self.padded_vocab_size = find_multiple(self.vocab_size, 64)
```

Comment:
We could throw a warning here if we want to. I thought it was unnecessary though because the user would already be doing the conscious choice of passing `padded_vocab_size=non_64_multiple_vocab_size`